docs/developer/releasing.md

```diff
@@ -62,26 +62,26 @@ export GPG_TTY=$(tty)
 
 ## Performing Releases
 
-The lifetime of a release branch is sheparded by a single person, picked from a
-rota of project maintainers and contributors. The release shepard is
+The lifetime of a release branch is shepherded by a single person, picked from a
+rota of project maintainers and contributors. The release shepherd is
 responsible for managing all releases that compose the release branch.
 
-For a new release branch, the release shepard will:
+For a **new release branch**, the release shepherd will:
 
 1. Gather consensus on which commit should be used as a base for the release
    branch.
 
-2. Create and push the release branch from the selected base commit.
+2. Create and push the release branch from the selected base commit. 
+
+   The name of the release branch should be the name of the stable version we intend 
+   to release, e.g. `release-v0.31`. Release branches do not contain the `-rc.N` release 
+   candidate suffix, i.e. there is no `release-v0.31-rc.0`.
 
 3. Create a PR to cherry-pick additional commits into the release branch as
    needed.
 
-4. Create a PR to [update code](#updating-code) for the upcoming release
-   candidate. A new section in the changelog should be added for the release
-   candidate, documenting all changes introduced by the release candidate.
-
-4. Update the changelog with a new section for the upcoming release candidate,
-   documenting changes that will be introduced in that release candidate.
+4. Create a PR to [update code](#updating-code) using the `-rc.N` release
```

Comment:
We should note that branches used for PRs should have a name that doesn't start with `release-`, otherwise branch protection rules apply to it. Alternatively, branches used for PRs can come from forks.

Reply:
Thank you, I'll add it like this:

```
4. Create a PR to [update code](#updating-code) using the `-rc.N` release
   candidate version.

   NOTE: PRs should have a name that doesn't start with `release-`, otherwise branch 
   protection rules apply to it. Alternatively, branches used for PRs can come from forks.
```